### ISSUE TITLE
packages not found error

### ISSUE THREAD

**Issue by pdrum** [matryer/moq#95]
I am getting this error when running moq on Mac with Go 1.11.6 (`moq . Hi` for example):

> Couldn't load source package: packages not found

Any idea why it is happening and how to fix it?

BTW this did not happen a while ago. some new release might have caused it if it's not sth I have done wrong in my own code.

**Comment by pdrum** [comment]
![image](https://user-images.githubusercontent.com/10900655/54930937-c9434200-4f35-11e9-96e6-f5fbce2e4046.png)

**Comment by pdrum** [comment]
I also ran tests of moq itself and they fail with following messages
There might be sth wrong with my environment but I can't figure out what
![image](https://user-images.githubusercontent.com/10900655/54931452-a7968a80-4f36-11e9-938b-5f7ec5c92cc0.png)

**Comment by pdrum** [comment]
After hours of searching I came across this issue https://github.com/google/wire/issues/125 in wire repo. It seems I had updated my go installation to 1.12 (then 1.11 to see if it helps) but my golang.org/x/tools was leftover from go 1.9.6 and golang.org/x/tools/go/packages which was left from previous installation was the reason for these errors. 

In order to fix it, I removed both go tools and moq, then reinstalled both and everything worked.
